plugins/hypervisors/kvm/src/main/java/com/cloud/hypervisor/kvm/resource/KVMHAVMActivityChecker.java

```diff
@@ -52,11 +52,14 @@ public Boolean checkingHB() {
         cmd.add("-t", String.valueOf(String.valueOf(System.currentTimeMillis() / 1000)));
         cmd.add("-d", String.valueOf(suspectTimeInSeconds));
         OutputInterpreter.OneLineParser parser = new OutputInterpreter.OneLineParser();
+
         String result = cmd.execute(parser);
-        LOG.debug("KVMHAVMActivityChecker pool: " + nfsStoragePool._poolIp);
-        LOG.debug("KVMHAVMActivityChecker result: " + result);
-        LOG.debug("KVMHAVMActivityChecker parser: " + parser.getLine());
-        if (result == null && parser.getLine().contains("DEAD")) {
+        String parsedLine = parser.getLine();
+
+        LOG.debug(String.format("Checking HB with KVMHAVMActivityChecker [{command=\"%s\", result: \"%s\", log: \"%s\", pool: \"%s\"}].", cmd.toString(), result, parsedLine, nfsStoragePool._poolIp));
```

Comment:
I assuming that HB should be HA:
1. Checking **HB** with KVMHAVMActivityChecker ...
2. Checking **HA** with ...

Reply:
Never mind, it is the Heart Beat, right?

I would suggest a different name then as HB might not mean much.

Reply:
@GabrielBrascher done.